specification/app/data-plane/Microsoft.App/preview/2023-08-01-preview/containerapps.json

```diff
@@ -0,0 +1,806 @@
+        "name": "ExecutionType",
+        "modelAsString": true,
+        "values": [
+          {
+            "name": "synchronous",
+            "value": "synchronous",
+            "description": "Synchronous"
+          }
+        ]
+      }
+    },
+    "FakedSharedModel": {
+      "type": "object",
+      "description": "Faked shared model",
+      "properties": {
+        "tag": {
+          "type": "string",
+          "description": "The tag."
+        },
+        "createdDate": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The created date."
+        }
+      },
+      "required": [
+        "tag",
+        "createdDate"
+      ]
+    },
+    "ResourceGroup": {
+      "type": "object",
+      "description": "Resource group resource.",
+      "properties": {
+        "resourceGroupName": {
+          "type": "string",
+          "description": "The name of the resource group. The name is case insensitive.",
+          "minLength": 1,
+          "maxLength": 90
+        }
+      },
+      "required": [
+        "resourceGroupName"
+      ]
+    },
+    "SessionPool": {
+      "type": "object",
+      "description": "Session pool resource.",
+      "properties": {
+        "sessionPoolName": {
+          "type": "string",
+          "description": "The name of the session pool."
+        }
+      },
+      "required": [
+        "sessionPoolName"
+      ]
+    },
+    "Subscription": {
+      "type": "object",
+      "description": "Subscription resource.",
+      "properties": {
+        "subscriptionId": {
+          "$ref": "#/definitions/Azure.Core.uuid",
+          "description": "The ID of the target subscription. The value must be an UUID."
+        }
+      },
+      "required": [
+        "subscriptionId"
+      ]
+    },
+    "Widget": {
+      "type": "object",
+      "description": "A widget.",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The widget name.",
+          "readOnly": true
+        },
+        "manufacturerId": {
+          "type": "string",
+          "description": "The ID of the manufacturer."
+        },
+        "sharedModel": {
+          "$ref": "#/definitions/FakedSharedModel",
+          "description": "The faked shared model."
+        }
+      },
+      "required": [
+        "name",
+        "manufacturerId"
+      ]
+    }
+  },
+  "parameters": {
+    "Azure.Core.Foundations.ApiVersionParameter": {
+      "name": "api-version",
+      "in": "query",
+      "description": "The API version to use for this operation.",
+      "required": true,
+      "type": "string",
+      "minLength": 1,
+      "x-ms-parameter-location": "method",
+      "x-ms-client-name": "apiVersion"
+    },
+    "SessionIdentifier": {
+      "name": "identifier",
+      "in": "query",
+      "description": "The identifier of the session.",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "method"
```

Comment:
Should be a client property?https://azure.github.io/autorest/extensions/#x-ms-parameter-location